lib/nimble_template/helpers/dependency.ex

```diff
@@ -1,5 +1,33 @@
 defmodule NimbleTemplate.DependencyHelper do
-  def fetch_and_install_dependencies(),
+  alias NimbleTemplate.Generator
+
+  def fetch_and_install_dependencies,
     do:
       if(Mix.shell().yes?("\nFetch and install dependencies?"), do: Mix.shell().cmd("mix deps.get"))
+
+  def order_dependencies! do
+    file_content = File.read!("mix.exs")
+
+    dependencies = extract_dependencies(file_content)
+
+    ordered_dependencies =
+      dependencies
+      |> String.split(",\n")
+      |> Enum.sort()
+      |> Enum.join(",\n")
+
+    Generator.replace_content("mix.exs", dependencies, ordered_dependencies)
+  end
+
+  defp extract_dependencies(contents) do
+    [_, deps_with_file_footer] =
+      :binary.split(contents, """
```

Comment:
@hanam1ni I'm curious can we use String module instead of using the erlang module 🤔

```
String.split

vs

:binary.split
```